File: app/src/main/java/hu/bme/szn/randomjokes/interactor/jokes/event/EditJokeEvent.java
package hu.bme.szn.randomjokes.interactor.jokes.event;

import hu.bme.szn.randomjokes.model.Joke;

public class EditJokeEvent {

    private Joke joke;
    private Throwable throwable;

    public EditJokeEvent() {
    }

    public EditJokeEvent(Joke joke, Throwable throwable) {
        this.joke = joke;
        this.throwable = throwable;
    }

    public Joke getJoke() {
        return joke;
    }

    public void setJoke(Joke joke) {
        this.joke = joke;
    }

    public Throwable getThrowable() {
        return throwable;
    }

    public void setThrowable(Throwable throwable) {
        this.throwable = throwable;
    }
}

File: app/src/main/java/hu/bme/szn/randomjokes/interactor/jokes/event/EditJokeEvent.kt
package hu.bme.szn.randomjokes.interactor.jokes.event

import hu.bme.szn.randomjokes.model.MyJoke

data class EditJokeEvent(
        var code: Int = 2,
        var joke:MyJoke? = null,
        var throwable: Throwable? = null
)